public/locales/en/component-meta.json

```diff
@@ -0,0 +1,64 @@
+{
+  "uikit": {
```

Comment:
We use flat JSON in localization files, nested fields can be a problem for automation.

`uikit.accordion` –> `uikit_accordion`